tests/e2e/specs/conference.spec.js

```diff
@@ -0,0 +1,24 @@
+// about cypress and iframes https://www.cypress.io/blog/2020/02/12/working-with-iframes-in-cypress/
+const getIframeDocument = () => cy
+  .get('#jitsiConferenceFrame0')
+  .its('0.contentDocument')
+  .should('exist');
+
+const getIframeBody = () => getIframeDocument()
+  .its('body').should('not.be.undefined')
+  .then(cy.wrap);
+
+describe('Conference.vue', () => {
+  before(() => {
+    cy.visit('/league');
+  });
+
+  it('change route by click to the iframe "OK" button', () => {
+    const ROOM = 'SOME_ROOM_NAME';
+
+    getIframeBody().find('#enter_room_field').type(ROOM);
+
+    getIframeBody().find('#enter_room_button').click();
+    cy.location('pathname').should('eq', `/league/${ROOM}`);
```

Comment:
I propose to use chaining more activally, like:
```suggestion
    cy.get('#jitsiConferenceFrame0')
     .its('0.contentDocument')
     .should('exist')
     .its('body')
     .should('not.be.undefined')
     .find('#enter_room_field')
     .type(ROOM)
     .find('#enter_room_button')
     .click()
     .location('pathname')
     .should('eq', `/league/${ROOM}`)
```

Reply:
Also, does it test stuff inside of iframe? Isn't it enough to just ensure that iframe has a proper url? (then we will be able to enable chromeWebSecurity back)

Reply:
I think we don't need to test things inside the iframe atm.

When the integrated wallet is merged we can extend the e2e to cover connectivity tests.

Reply:
@mradkov maybe I can create a separate branch and PR for the test propose `just ensure that iframe has a proper url` and leave this one as `connectivity tests` and future?

Reply:
@mradkov I started it test `change iframe path by Vue route` in separate branch and will open a new PR. I think it's OK?